src/world/area_mac/mac_04/843030.c

```diff
@@ -80,7 +80,17 @@ INCLUDE_ASM(s32, "world/area_mac/mac_04/843030", func_80241A64_844634);
 
 INCLUDE_ASM(s32, "world/area_mac/mac_04/843030", func_80241B90_844760);
 
-INCLUDE_ASM(s32, "world/area_mac/mac_04/843030", func_80241BB8_844788);
+ApiStatus N(SetNpcShadowScale)(Evt* script, s32 isInitialCall) {
+    s32 npcID;
+    f32 newShadowScale;
+
+    Bytecode* args = script->ptrReadPos;
```

Comment:
args at the top of the func, and then npcID on the same line it's declared, followed by newShadowScale